packages/fhir-converter/src/templates/cda/Utils/GeneratePractitionerId.hbs

```diff
@@ -1,15 +1,11 @@
-
+{{!-- Id composed of root and extension if they both exist, else name addr telecom --}}
 {{#with (elementAt (toArray obj.id) 0) as |FirstId|}}
     {
         "Id":
             {{#if (and FirstId.root FirstId.extension)}}
                 "{{generateUUID (concat (toString FirstId.root) '|' (toString FirstId.extension))}}"
-            {{else if FirstId.root}}
-                {{!--If only root is provided, it is supposed to be unique--}}
-                "{{generateUUID (toString FirstId.root)}}"
-            {{else}}
-                {{!--Generate using obj if no root or id is available--}}
-                "{{generateUUID (toJsonString ../obj)}}"
+            {{else if ../obj.assignedPerson.name}}
+                "{{generateUUID (concat ../obj.assignedPerson.name ../obj.addr ../obj.telecom)}}"
```

Comment:
updating practitioner id. If there is no extension, the extension alone is useless. 
And then if there is no id but there is valuable demographic data (minimum name), then we will create a name out of that